user-interface/src/lib/components/uswds/Table.test.tsx

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
@fmaddenflx , note that I did verify that adding a violation of a different jsx-a11y rule triggered an error, so I'm assuming this was added during initial implementation but is no longer necessary. The same applies for a similar change Modal.tsx. Please let me know if these should be added back.